src/rez/rezconfig.py

```diff
@@ -269,6 +269,9 @@
 # Enable package caching during a package build.
 package_cache_during_build = False
 
+# Enable package caching to run asynchronously during a resolve.
```

Comment:
I find this hard to read. How about "set to false to always cache package payloads synchronously"?